### PR TITLE
Remove goal progress count from goal page

### PR DESCRIPTION
**Pull request by achidu**
Removed the progress count from the goal page. This was an artifact of when we didn't group goals and tasks into "Open" and "Completed" sections, and it didn't make sense as-is anymore.

I didn't remove the helper function in case we want it later — not sure if that's the right move.

### CODE REVIEW

**Comment by achidu** [comment]
https://app.shortcut.com/capable/story/7914/app-remove-x-x-complete-label-from-task-and-goal-list

File: src/components/GoalCards.js [hunk 1]
```diff
@@ -106,7 +106,6 @@ const GoalCards = ({ goals, status }) => {
           variant="subtitle"
           sx={{ display: "flex", gap: 1, fontSize: "0.88rem" }}
         >
-          {progressString(goals)}
```
Comment (lawyu89) [inline src/components/GoalCards.js:109]: may as well remove the whole <Typography> wrapping this line.

**Review by lawyu89 — CHANGES_REQUESTED** [review]
Looks good.

File: src/components/GoalCards.js (hunk 1 above)
Comment (achidu) [reply src/components/GoalCards.js:109]: oh right 😬 removed

**Review by lawyu89 — APPROVED** [review]

**Review by lvancraen — APPROVED** [review]
You don't need to, but I'd be of the opinion to remove the helper function. My guess is we won't use it for some time and when we do, it's not even a guarantee it works anymore. For something so simple, it may be easier to write a function from scratch then needing to refactor it.

**Pull request merged**